runbot_gitlab_minimal/models/runbot_repo.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+import os
+import logging
+import re
+import urllib
+
+import requests
+
+from openerp import models, fields
+
+_logger = logging.getLogger(__name__)
+
+
+class RunbotRepo(models.Model):
+    _inherit = "runbot.repo"
+
+    uses_gitlab = fields.Boolean(string='Use Gitlab', help='Use Gitlab')
```

Comment:
If the help use the same string of the string then delete the help or add a better one